careers/README.md

```diff
@@ -0,0 +1,53 @@
+
+## Open Opportunities
+
+We're currently looking to recruit the following roles:
+
+- [Senior Fullstack Node Engineer](jobs/senior-fullstack-node-engineer.md)
+
+## Hiring Process
+
+### Apply
+
+If you want to apply for any of the positions, please email [careers@dvelp.co.uk](mailto:careers@dvelp.co.uk) with your CV - we can't wait to hear from you!
+
+### Interviews
+
+Once your application has been reviewed, there will be a series of interviews to make sure we're the right fit for you, and vice versa. These are done one-at-a-time, with you progressing to the next if successful in the previous. We will try to keep these as close together as possible, and work with you to find appropriate times, be those in or out of working hours - we know it's not easy to get time off during the day for this!
+
+#### Short Telephone Chat (15mins)
+
+This is just a quick chat to talk informally so you can get to know us a bit better, and we can get to know a little more about you. We'll also get most of the "boring" questions out the way here, such as notice period, current salary, location and what not. We'll be asking why you want to work with us, and what excites you about a possible career at DVELP.
+
+#### Culture Fit Video call (45mins)
+
+During the culture fit call, we want to learn all about what makes you tick, and how you like to work. We'll cover things like teamwork, leadership, and what do you do when things go wrong. If you've got things you're passionate about in terms of process, now's the time to let us know! We all grow when we share ideas, and we want to learn what your thoughts are.
+
+#### Technical Video call (45mins)
+
+By now we're getting to know each other well, so it's time to dive into the nitty-gritty of what you'll be doing on a day-to-day, and how you've achieved it previously - what you've learned, and what you'll apply here. As an example for engineers, expect questions around how you would design systems (don't worry, we're not expecting encyclopedic knowledge).
+
+#### _Engineers Only:_ Programming Session (Up to 4hours)
+
+Whiteboard interviews are rubbish, so there's no chance we'd do them here at DVELP. What we do instead is invite you to spend up to 4 hours with one of our own engineers solving a problem. You'll be on a video chat sharing your screen, and building a solution. Our engineer isn't just there to judge though - they're on hand to help you out and guide you along the right path. Don't worry if you have to Google something every 10 seconds, we all do that too!
+
+We realise that trying to find 4 hours during your week can be hard, so we will try our best to accommodate you and can trim the time accordingly, and can run into outside office hours if needed. If the phone or doorbell rings or there's any interruptions, that's absolutely fine. We're used to working from home and all the distractions that can bring, so don't fret.
```

Comment:
`best to accommodate you and can trim the time accordingly, and can run into outside office hours if needed.` -> `best to accommodate you and can trim the time accordingly. We can run into outside office hours if needed.`